File: openapi/code_samples/python/api_v1_passwordless_email_verify/post.py
# Verify with OTP code
verify_response = scalekit_client.passwordless.verify_passwordless_email(
    code="123456",  # OTP code received via email
    auth_request_id=auth_request_id,
)

# Verify with magic link token
verify_response = scalekit_client.passwordless.verify_passwordless_email(
    link_token=link_token,  # Magic link token from URL
)

# User verified successfully
user_email = verify_response[0].email

File: openapi/code_samples/java/api_v1_passwordless_email_verify/post.java
// Verify with OTP code
VerifyPasswordlessOptions verifyOptions = new VerifyPasswordlessOptions();
verifyOptions.setCode("123456"); // OTP code
verifyOptions.setAuthRequestId(authRequestId);

VerifyPasswordLessResponse verifyResponse = scalekitClient.passwordless()
    .verifyPasswordlessEmail(verifyOptions);

// User verified successfully
String userEmail = verifyResponse.getEmail();

// Verify with magic link token
VerifyPasswordlessOptions linkVerifyOptions = new VerifyPasswordlessOptions();
linkVerifyOptions.setLinkToken(linkToken); // Magic link token

VerifyPasswordLessResponse linkVerifyResponse = scalekitClient.passwordless()
    .verifyPasswordlessEmail(linkVerifyOptions);

String verifiedEmail = linkVerifyResponse.getEmail();
